core/src/main/scala/org/bitcoins/core/protocol/BtcHumanReadablePart.scala

```diff
@@ -53,6 +53,7 @@ object BtcHumanReadablePart extends StringFactory[BtcHumanReadablePart] {
       case _: MainNet  => bc
       case _: TestNet3 => tb
       case _: RegTest  => bcrt
+      case _: SigNet   => tb
```

Comment:
:-(, they didn't give signet a unique prefix ?

Reply:
Initially we did, but apparently some people want all test nets to use the same prefix. Even regtest and testnet.